evergreen.yml

```diff
@@ -1960,7 +2052,44 @@ tasks:
   - name: semgrep
     exec_timeout_secs: 3600 # 1h
     commands:
-      - func: "static code analysis"
+      - func: "generate static code analysis"
+
+  - name: ssdlc-artifacts-release
```

Comment:
This will never be called because it is not tied to a variant.
It should go in the `release` variant.